src/components/course/data/hooks.jsx

```diff
@@ -338,14 +331,13 @@ useCoursePriceForUserSubsidy.propTypes = {
 export const useCourseEnrollmentUrl = ({
   enterpriseConfig,
   key,
-  location,
   couponCodes = [],
   sku,
   subscriptionLicense,
   userSubsidyApplicableToCourse,
 }) => {
   const config = getConfig();
-  const baseQueryParams = new URLSearchParams(location.search);
+  const baseQueryParams = new URLSearchParams(global.location.search);
```

Comment:
nit: Should we get `location` from `react-router-dom`'s `useLocation`?

Reply:
yep!